views/index.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
If you wanted to keep a root view it probably would be better to rename it `root` instead. Another alternative would be redirect users visiting `'/'` to another existing view like `apartments/index`